config/build.conf.js

```diff
@@ -41,6 +43,13 @@ const ANALYTICS_CONFIG = path.join(MONGOSH, 'lib', 'analytics-config.js');
  */
 const APPLE_NOTARIZATION_BUNDLE_ID = 'com.mongodb.mongosh';
 
+/**
+ * The SHA for the current git HEAD.
+ */
+const REVISION = process.env.IS_PATCH ?
+  `pr-${process.env.GITHUB_PR_NUMBER}-${process.env.REVISION_ORDER_ID}` :
```

Comment:
How is this used? This will be probably super-useful for releases too.

Reply:
Err, it might be easier to show this once CI has run :) It’s used for uploading artifacts to S3; e.g. the one from this PR would now become available at 

https://s3.amazonaws.com/mciuploads/mongosh/pr-433-anna.henningsen_158/mongosh-0.5.2-linux.tgz

instead of 

https://s3.amazonaws.com/mciuploads/mongosh/dde7b239f0cef50d217a4bf1766a89c8890a2e9d/mongosh-0.5.2-linux.tgz

(where dde7b239f0cef50d217a4bf1766a89c8890a2e9d is the *base* revision, not a commit from this PR). That matters because we would otherwise overwrite the already-existing tarball for dde7b239f0cef50d217a4bf1766a89c8890a2e9d – we’re lucky we didn’t have problems with this, but in the worst case, I think this could previously have potentially made builds from a PR override a release build :grimacing: